Libraries/Components/MapView/MapView.js

```diff
@@ -303,6 +309,7 @@ const MapView = React.createClass({
     onAnnotationPress: React.PropTypes.func,
 
     /**
+     * Unused - Android version of MapView not currently available.
```

Comment:
Engineers at Facebook can actually use the internal Android implementation and could get confused by this. Might be better to keep as is, given there's a big warning in the docs at the top (thanks for adding it!).